charters/DataPortal/charter.md

```diff
@@ -0,0 +1,80 @@
+# Data Portal
+
+
+## Description
+The Data Portal is a landing site for the Human Cell Atlas (HCA) Data Coordination Platform (DCP). It is hosted at https://www.data.humancellatlas.org/ with an access link from the main HCA website at https://www.humancellatlas.org/. It provides the user experience for scientific and general users to upload, explore, search, and run analyses on human genomic data. The Data Browser ("Explore") application provides a faceted search capability to find and download subsets of this data.
+
```

Comment:
There seems to be no concept of ensuring the community can utilise the HCA data to deliver scientific value. This seems to be a significant oversight

Reply:
> on human genomic data.

Perhaps this should be worded more like the data portal homepage and say "single-cell data" or "cellular resolution data". Currently, the DCP *does* accept mouse data and while we accept mostly genomic data now, in the future we might also accept proteomic and other non-genomic data.

Reply:
Cellular resolution data seems like a good plan

Reply:
This is a bit misleading as the data portal itself will not be handling the user upload experience. Also, will it really be possible to run analyses from the dataportal?

Reply:
I've adopted the "cellular  resolution data" suggestion and removed the references to "upload" and "running analyses".
I'm open to suggestions as to how to express the concept of "ensuring the community can utilise the HCA data to deliver scientific value".

Reply:
Both the metadata and ingest support charters have statements in the objects like this 

`Ensuring the defined metadata schema provides a positive user experience for those who read and write them`

`Working with the UX team to ensure the submission process provides a positive user experience to data contributors`

These provide explicitly statements that usability is at the forefront of our concerns rather than an after thought